src/components/TopBar.js

```diff
@@ -107,10 +115,69 @@ export default Utils.connectAtom(authStore, 'authState')(class TopBar extends Co
       onClick: () => this.hideNav()
     }, [
       div({ style: styles.nav.icon }, [
-        icon(iconName, { className: 'is-solid', size: 24 })
+        icon(iconName, {
+          className: 'is-solid',
+          size: 24
+        })
       ]),
       label
     ])
+
+    const enabledCredits = h(Clickable, {
+      style: {
+        ...styles.nav.item,
+        borderBottom: 'none'
+      },
+      hover: { backgroundColor: colors.darkBlue[1] },
+      onClick: () => this.setState({ openFreeCreditsModal: true })
+    }, [
+      div({ style: styles.nav.icon }, [
+        icon('cloud', {
+          className: 'is-solid',
+          size: 20
+        })
+      ]),
+      'Sign up for $300 in free cloud credits'
+    ])
+
+    const enrolledCredits = h(Clickable, {
+      style: {
+        ...styles.nav.item,
+        borderBottom: 'none'
+      },
+      as: 'a',
+      hover: { backgroundColor: colors.darkBlue[1] },
+      href: 'https://software.broadinstitute.org/firecloud/documentation/freecredits',
+      target: '_blank',
+      onClick: () => this.hideNav()
+    }, [
+      div({ style: styles.nav.icon }, [
+        icon('cloud', {
+          className: 'is-solid',
+          size: 20
+        })
+      ]),
+      'You are accessing your free cloud credits',
```

Comment:
I think the link text should be a little shorter, like "You have free credits"
![screen shot 2019-01-31 at 9 42 11 am](https://user-images.githubusercontent.com/2395211/52061578-9afc4400-253c-11e9-99de-aa11feb09cce.png)
Or "Access free credits"
![screen shot 2019-01-31 at 9 43 51 am](https://user-images.githubusercontent.com/2395211/52061620-b9fad600-253c-11e9-8654-b5e3fa37423f.png)

Note if you shorten the text that you'll need to add a little padding or margin between the text and the external link icon

Reply:
I think the external link icon is inappropriate here because the link does not navigate away from Terra.